rds-deployment-ha-broker/Scripts/RdcbHaPostConfig.ps1

```diff
@@ -20,7 +20,7 @@ param
      [string]$clientAccessName,
      [string]$sqlClientUrl,
      [string]$dnsServer,
-     [string]$sqlServer
+     [string]$sqlServer=""
```

Comment:
just set it default as empty string. But it didn't fix the issue when the parameter is not populated. What I did is the variable section in the template to put quote around the parameter even it is empty, the parameter will still pass in to prevent the issue. it works for me